Installing-Oppia-(Mac-OS;-Python-3).md

```diff
@@ -19,11 +19,17 @@ To check whether your Mac has an M1 chip, navigate to the Apple menu and select
 
 If your Mac has an M1 chip, follow these instructions instead:
 
-1. [Install](https://stackoverflow.com/a/64883440) Rosetta 2
+1. If not yet installed, install Rosetta 2 with the following command:  `softwareupdate --install-rosetta`. Rosetta 2 translates Intel-based apps to run on Apple silicon Macs.
 
-2. Inside Rosetta perform the Downloading and prerequisites steps (**Note:** If `sudo easy_install pyyaml` does not work try using `pip3 install pyyaml`).
+2. Next, we'll create two aliases to make it more convenient to switch between Mac M1s' *ARM* architecture and the emulated *Intel* architecture. To do so, run the following in the terminal (change "zsh" to "bash" if you're using a bash terminal):
+    ```shell
+    $ alias arm="env /usr/bin/arch -arm64 /bin/zsh --login"
+    $ alias intel="env /usr/bin/arch -x86_64 /bin/zsh --login"
+    ``` 
+    Now, run `intel` in the terminal followed by `arch`. You should see `i386` being printed. This is the architecture we'll use for this installation process. (**Note:** To change back to ARM64, run "arm" or simply open a new terminal).
```

Comment:
Running `arm` here will open a new shell inside the existing shell, so it's not quite switching. This might be important because if users run arm/intel a lot, they'll make a lot of nested shells. You can see this by running `exit` after you call arm/intel a few times--you'll have to run `exit` multiple times to exit out of all the nested shells